janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -257,6 +265,31 @@ public void register(String store, String key, KeyInformation information, BaseT
             }
         }
         //Since all data types must be defined in the schema.xml, pre-registering a type does not work
+        //But we check Analyse feature
```

Comment:
To fail during the property registration and not during the search. So the index will not have property with bad configuration.
I think if the configuration is wrong, JanusGraph should fail as soon as possible.